daemon/nextbox_daemon/jobs.py

```diff
@@ -452,10 +452,38 @@ def _run(self, cfg, board, kwargs):
         DockerControl().purge_old_images()
 
 
+class OccUpgradeJob(BaseJob):
+    """
+    Job to periodically purge old images using
```

Comment:
please update the docstring to match the job

Reply:
done